### DISCUSSION TITLE
.editorconfig

### DISCUSSION THREAD

**Discussion by daiyam** [4ian/GDevelop#4561] [Enhancement ideas (and their technical discussions)]
Hello,

I'm new to GDevelop and I was wondering if it was ok to make a PR to add the [.editorconfig](https://editorconfig.org/) file?

> EditorConfig helps maintain consistent coding styles for multiple developers working on the same project across various editors and IDEs.

**Comment by AlexandreSi** [comment]
Is it to be able to format javascript code events as configured in this .editorconfig file? Or do you have something else in mind?

**Reply by daiyam** [reply]
[EditorConfig](https://editorconfig.org/) is only used to manage the indent (size & characters) and the newlines (characters, trim, end of file). But it will work on bunch of editors and all text files.
